Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -49,3 +49,40 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### Spec
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
+
+
+## Creating a Storage Class
+
+Once the RADOS namespace is created, an RBD-based StorageClass can be created to
+create PVs in this RADOS namespace. For this purpose, the `clusterID` value from the
+CephBlockPoolRadosNamespace status needs to be put into the `clusterID` field of the StorageClass
+spec. 
+
+Extract the clusterID from the CephBlockPoolRadosNamespace CR:
+
+ ```shell
+ $ kubectl -n rook-ceph get cephblockpoolradosnamespace -o yaml
+  ...
+  status:
+    info:
+      clusterID: 80fc4f4bacc064be641633e6ed25ba7e
+```
+
+Now set the `clusterID` retrieved from the previous step into the `clusterID` of the storage class.
+For a full example, see deploy/examples/csi/rbd/storageclass-radosnamespace.yaml.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-ceph-block
+provisioner: rook-ceph.rbd.csi.ceph.com # csi-provisioner-name
+parameters:
+  clusterID: 80fc4f4bacc064be641633e6ed25ba7e
+```
+
+
+
+
+
+
```

Comment:
markdownlint should be catching this. AFAICT, this PR is based on an out of date fork that doesn't have the markdownlint github action configured. @obnoxxx please pull the latest Rook upstream changes and rebase on top.